src/App/Resources/AppResources.resx

```diff
@@ -2220,4 +2220,19 @@
   <data name="SpecialCharacters" xml:space="preserve">
     <value>Special Characters (!@#$%^&amp;*)</value>
   </data>
+  <data name="FilterByVault" xml:space="preserve">
+    <value>Filter items by vault</value>
+  </data>
+  <data name="AllVaults" xml:space="preserve">
+    <value>All Vaults</value>
+  </data>
+  <data name="Vaults" xml:space="preserve">
+    <value>Vaults</value>
+  </data>
+  <data name="VaultFilterDescription" xml:space="preserve">
+    <value>Vault: {0}</value>
+  </data>
+  <data name="VaultFilterDescriptionAll" xml:space="preserve">
+    <value>All</value>
+  </data>
```

Comment:
Shouldn't we just use `All` as the key here? I think it's more 1:1 reference and easier to know the content of the string.